lib/appsignal/config.rb

```diff
@@ -269,8 +294,10 @@ def load_config
       @initial_config[:env] = @env
 
       # Load the config file if it exists
-      @file_config = load_from_disk || {}
-      merge(file_config)
+      if @load_yaml_file
+        @file_config = load_from_disk || {}
+        merge(file_config)
+      end
```

Comment:
Could we emit a warning whenever one of the two files does exist but it's intentionally not being loaded? (For example, here, when `@load_yaml_file` is `false`, but also in other places)

This might help users debug why their `.yml` config is being ignored.

Reply:
I've added some warnings in [this commit](https://github.com/appsignal/appsignal-ruby/pull/1324/commits/cdbec613ec43bd469f70cf637665b0f16451e964)